types/time.go

```diff
@@ -118,31 +118,19 @@ var (
 	ZeroDuration = Duration{Duration: gotime.Duration(0), Fsp: DefaultFsp}
 
 	// ZeroCoreTime is the zero value for TimeInternal type.
-	ZeroCoreTime = MysqlTime{}
+	ZeroCoreTime = Time(0)
 
 	// ZeroCoreTime is the zero value for Time type.
-	ZeroTime = Time{}
+	ZeroTime = Time(0)
```

Comment:
Do we need to distinct the `ZeroTime` and `ZeroCoreTime` even if they are same?

Reply:
We still have `SetCoreTime` and `CoreTime` API now, so export `ZeroCoreTime` is reasonable.

BTW, I think that we can keep the API compatibility in this PR to reduce the diff.